one-page.js

```diff
@@ -38,6 +38,13 @@ function inlineAssets(projectPath) {
             };
 
             (function () {
+               // Patch the http get function to check for an object being passed to it and return immediately
```

Comment:
minor detail, can we align indentation here

Reply:
Done!